### DISCUSSION TITLE
cmake-tools plugin

### DISCUSSION THREAD

**Discussion by deveezy** [fitrh/init.nvim#10] [General]
Hi, i'm trying to embed Civitasv/cmake-tools.nvim plugin to your setup, but something went wrong.

config/plugin/cmake-tools/packerspec.lua:

```lua
return {
  "Civitasv/cmake-tools.nvim",
  config = function()
    require("config.plugin.cmake-tools.setup")
  end,
}
```

config/plugin/cmake-tools/setup.lua:

```lua
require("cmake-tools").setup {
  ...
}
```
config/plugin/packer.lua:
```lua
require("config.plugin.cmake-tools.packerspec"),
```

My nvim not showing any errors, but i can't do anything after open something.

Can you help pls?)

**Comment by fitrh** [comment]
You need to run the `PackerInstall` command after adding new plugins, either by running

```sh
./script/packer/install.sh
```

or manually open neovim and run the following commands

1. `:lua require('config.plugin.packer')`
2. `:PackerInstall`

Packer may warn you about unused plugins, just ignore them by pressing `Enter`.

Sorry for the inconvenience, I haven't had time to update the README.

**Comment by deveezy** [comment]
The thing is, i already did it for other plugins. It works good. But for this plugin something went wrong. More accurately about problem: when i run nvim i can't use any telescope keybinds and no commands CMakeRun, CMakeGenerate etc, But other plugin works fine. If i remove keys = {} from telescope packerspec then recompile packer cmake plugin works

**Reply by deveezy** [reply]
Now i've added this line : ```event = "BufRead",``` to packerspec file and now it works
